tests/ragger/conftest.py

```diff
@@ -1,4 +1,53 @@
+import sys
+from pathlib import Path
+import warnings
+import glob
+
 from ragger.conftest import configuration
 
+#######################
+# CONFIGURATION START #
+#######################
+
+# You can configure optional parameters by overriding the value of
+# ragger.configuration.OPTIONAL_CONFIGURATION
+# Please refer to ragger/conftest/configuration.py for their descriptions and accepted values
+
+def pytest_addoption(parser):
+    parser.addoption("--with_lib_mode", action="store_true", help="Run the test with Library Mode")
+
+
+parent: Path = Path(__file__).parent.parent
```

Comment:
Since in the workflow it has been moved one folder down to `tests/ragger`, only one `.parent` is needed here now.

We should also probably push this directory (`tests/ragger/.test_dependencies/clone/`) so it does not have to be re-created for local-use, like [how it is done in plugin-boilerplate](https://github.com/LedgerHQ/app-plugin-boilerplate/tree/develop/tests/.test_dependencies/ethereum) with an empty file inside so Git lets-us commit it. :+1:

Reply:
Fixed